README.md

```diff
@@ -75,7 +75,7 @@ $profile = new KlaviyoProfile(
     )
 );
 
-$client->publicAPI->identify( $profile );
+$client->publicAPI->identify( $profile, $post=true );
```

Comment:
one last thing, unless someone is on php 8, named arguments aren't supported so we can just do:
```suggestion
$client->publicAPI->identify( $profile, true );
```